From 155a22ab58635b89a14d9871543e341f17893c5f Mon Sep 17 00:00:00 2001
Message-Id: <155a22ab58635b89a14d9871543e341f17893c5f.1358959439.git.minovotn@redhat.com>
In-Reply-To: <4b2df53c087cd9df02d66686da40ae4f600f3904.1358959439.git.minovotn@redhat.com>
References: <4b2df53c087cd9df02d66686da40ae4f600f3904.1358959439.git.minovotn@redhat.com>
From: Markus Armbruster <armbru@redhat.com>
Date: Wed, 23 Jan 2013 15:52:32 +0100
Subject: [PATCH 6/7] Revert "audio: add VOICE_VOLUME ctl"

RH-Author: Markus Armbruster <armbru@redhat.com>
Message-id: <1358956353-15195-7-git-send-email-armbru@redhat.com>
Patchwork-id: 47669
O-Subject: [RHEL-6.4 PATCH qemu-kvm 6/7] Revert "audio: add VOICE_VOLUME ctl"
Bugzilla: 884253
RH-Acked-by: Ademar de Souza Reis Jr. <areis@redhat.com>
RH-Acked-by: Alex Williamson <alex.williamson@redhat.com>
RH-Acked-by: Gerd Hoffmann <kraxel@redhat.com>

This reverts commit 555212a9e981111d6df337beab02fe59f81d9860.

Signed-off-by: Markus Armbruster <armbru@redhat.com>
---
 audio/audio.c     | 12 ------------
 audio/audio_int.h |  1 -
 2 files changed, 13 deletions(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 audio/audio.c     | 12 ------------
 audio/audio_int.h |  1 -
 2 files changed, 13 deletions(-)

diff --git a/audio/audio.c b/audio/audio.c
index 565248e..89e618f 100644
--- a/audio/audio.c
+++ b/audio/audio.c
@@ -2045,29 +2045,17 @@ void AUD_del_capture (CaptureVoiceOut *cap, void *cb_opaque)
 void AUD_set_volume_out (SWVoiceOut *sw, int mute, uint8_t lvol, uint8_t rvol)
 {
     if (sw) {
-        HWVoiceOut *hw = sw->hw;
-
         sw->vol.mute = mute;
         sw->vol.l = nominal_volume.l * lvol / 255;
         sw->vol.r = nominal_volume.r * rvol / 255;
-
-        if (hw->pcm_ops->ctl_out) {
-            hw->pcm_ops->ctl_out (hw, VOICE_VOLUME, sw);
-        }
     }
 }
 
 void AUD_set_volume_in (SWVoiceIn *sw, int mute, uint8_t lvol, uint8_t rvol)
 {
     if (sw) {
-        HWVoiceIn *hw = sw->hw;
-
         sw->vol.mute = mute;
         sw->vol.l = nominal_volume.l * lvol / 255;
         sw->vol.r = nominal_volume.r * rvol / 255;
-
-        if (hw->pcm_ops->ctl_in) {
-            hw->pcm_ops->ctl_in (hw, VOICE_VOLUME, sw);
-        }
     }
 }
diff --git a/audio/audio_int.h b/audio/audio_int.h
index 117f95e..2003f8b 100644
--- a/audio/audio_int.h
+++ b/audio/audio_int.h
@@ -231,7 +231,6 @@ void audio_run (const char *msg);
 
 #define VOICE_ENABLE 1
 #define VOICE_DISABLE 2
-#define VOICE_VOLUME 3
 
 static inline int audio_ring_dist (int dst, int src, int len)
 {
-- 
1.7.11.7

